my-app/src/App.tsx

```diff
@@ -1,8 +1,94 @@
 import React from "react";
 import "./App.css";
+import { createTheme, colors } from "@mui/material";
+import { ThemeProvider } from "@mui/system";
+import { LocalizatorProvider } from "@mui/lab";
+import AdapterDateFns from "@mui/lab/AdapterDateFns";
+import { spacing } from '@mui/system'
+
+const theme = createTheme({
+    breakpoints: {
+      values: {
+        xs: 0,
+        sm: 480,
+        md: 767,
+        lg: 1200,
+        xl: 1519,
+      },
+
+      spacing: { 
+        xs: 0,
+        sm: 1.8,
+        md: 5,
+        lg: 10.25,
+        xl: 12.5,
+      
+
+  },
+},
+  
+  palette: {
+    primary: {
+      main: '#f48982',
+    },
+    secondary: {
+      main: '#fbdb89',
+    },
+
+    info:{
+      main:'#918581'
+    },
+
+    background:{
+      main:"#ffff"
+      darker:"#f9f5f3"
+    }
+
+    typography:{
+      fontFamily:['Nunito Sans, sans-serif'],
+    
+      h1:{
+      fontSize: "10px",
+      color:"#f9f5f3",
+      fontWeight:"400",
+      lineHeight: "16px"
+      letterSpacing:"1.6px"
+      }
+
+      h2: {
+        fontSize: "32.5px",
+        color:'#f48982',
+        fontWeight:"700",
+        lineHeight: "63.375px"
```

Comment:
Check all line heights, my advice is to not define it at all, let it be auto. 
This is for all typography.